source/nvda.pyw

```diff
@@ -251,7 +256,9 @@ if customVenvDetected:
 	log.warning("NVDA launched using a custom Python virtual environment.")
 if globalVars.appArgs.changeScreenReaderFlag:
 	winUser.setSystemScreenReaderFlag(True)
-#Accept wm_quit from other processes, even if running with higher privilages
+# Accept wm_quit from other processes, even if running with higher privilages
+# This allows for downgrading between NVDA versions, deprecated in 2022.1
+# in 2022.1 remove the call to ChangeWindowMessageFilter (#12286)

```

Comment:
I'd say some thing like "2020.4 and earlier versions sent a WM_QUIT message when asking NVDA to exit. Some users may run several different versions of NVDA, so we continue to support this. "

I don't think we need to mark it as deprecated, or have a plan for removal at this stage. I wouldn't consider removing this for many years.

If it is still true, you might also add "WM_QUIT does not allow NVDA to shutdown cleanly, now WM_EXIT_NVDA is used instead".